pytorch_lightning/core/memory.py

```diff
@@ -3,14 +3,14 @@
 '''
 
 import gc
-import logging as log
 import os
 import subprocess
 from subprocess import PIPE
 
 import numpy as np
 import torch
 
+from ..core import root_logger as log
```

Comment:
the rest of the code base uses absolute imports. maybe change it to be consistent.